riscv/decode.h

```diff
@@ -7,6 +7,7 @@
 # error spike requires a two''s-complement c++ implementation
 #endif
 
+#include "config.h"
```

Comment:
Adding include `config.h` here breaks libraries which want to include spike's installed headers.
`decode.h` is an installed header, while `decode_macros.h` is not.

Is it possible to move the int128 typedefs to `decode_macros.h`, or `v_ext_macros.h`?

Reply:
Oh sorry, completely missed this. Let me have a look at this.

Reply:
Updated, thanks!